components/stepper/steps/overview.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Overview
+page_title: Overview
+description: Steps of the Stepper for Blazor.
+slug: stepper-steps-overview
+tags: telerik,blazor,steps,overview
+published: True
+position: 0
+---
+
+# Steps Overview
+
+The Steps are the building blocks of the Stepper component. You can include the desired steps by adding a  `StepperStep` tag for every step. The steps appearance can be configured through the features the `StepperStep` exposes or with a [Step Template]({%slug step-template%})
+
+In this article:
+
+* [Steps Declaration](#steps-declaration)
+* [Steps Features](#features)
+
+## Steps Declaration
+
+To include the desired steps for the Stepper component, you can either manually declare a `StepperStep` tag for every step or loop through your collection with steps data and render one `StepperStep` tag binding its parameters to the corresponding field of your model.
+
+>caption Loop through your collection and generate a `StepperStep` for every item in the collection. The result from the snippet.
+
+![Steps collection](images/steps-collection-example.png)
+
+````CSHTML
+@*Loop through a collection to create a step for all items in the collection. *@
+
+<div style="width:700px">
+    <TelerikStepper>
+        <StepperSteps>
+            @foreach (var step in Steps)
+            {
+            <StepperStep Label="@step.Label" Icon="@step.Icon" 
+                         Optional="@step.Optional" Disabled="@step.Disabled">
+            </StepperStep>
+            }
+        </StepperSteps>
+    </TelerikStepper>
+</div>
+
+@code{
+    List<StepModel> Steps { get; set; }
+
+    protected override void OnInitialized()
+    {
+        Steps = new List<StepModel>()
+    {
+            new StepModel()
+            {
+                Label = "Personal Info",
+                Icon = "user"
+            },
+            new StepModel()
+            {
+                Label = "Education",
+                Icon = "dictionary-add",
+                Disabled = true
+            },
+            new StepModel()
+            {
+                Label = "Experience",
+                Icon = "flip-vertical"
+            },
+            new StepModel()
+            {
+                Label = "Attachments",
+                Icon = "attachment",
+                Optional = true
+            }
+        };
+            base.OnInitialized();
+    }
+
+    public class StepModel
+    {
+        public string Label { get; set; }
+        public string Icon { get; set; }
+        public bool Disabled { get; set; }
+        public bool Optional { get; set; }
+    }
+}
+````
+
+## Features
+
+The `StepperStep` exposes the following features which allow you to configure their appearance:
+
+* [Indicators]({%slug stepper-indicators%})
```

Comment:
If the details for those parameters are given in the specific articles, consider either dropping this content form there, or templating the list to make sure it is easier to maintain in the future.

Reply:
I wanted to keep this as a summary for the steps features, I am not sure if I can template it as all the information is spread and included in separate sections in the corresponding articles.
May be I can just leave the bullets with no info for them but links to the articles? Please advise